### ISSUE TITLE
Send a message based on the title of the marketplace post

### ISSUE THREAD

**Issue by effeci** [carpedm20/fbchat#557]
## Description of the problem
Example: Logging in fails when the character `%` is in the password. A specific password that fails is `a_password_with_%`

# Example code
import fbchat
from fbchat.models import *

# Subclass fbchat.Client and override required methods
class EchoBot(fbchat.Client):
    def onMessage(self, author_id, message_object, thread_id, thread_type, **kwargs):
        # Works with both groups and user chats, if you want only groups, check `thread_type is ThreadType.GROUP`
        print("{} from {}".format(message_object, thread_id))
        # If the message matches
        if message_object.text == "POST TITLE":
            # Do something, e.g. send a message back
            self.send(Message(text="hello...."), thread_id=thread_id, thread_type=thread_type)
    
bot = EchoBot(“user”, “pass”)
bot.listen()

So basically this is echobot example. The problem is that when someone texts me in for a marketplace post, I have to give a different answer based on the product
From the listening in the terminal I see this type of chat as a group and asap the chat is opened the thread title gets changed to the name of the product. Now, where is this name stored? How can I parse if it's the text i'm looking for to give a certain answer? Thanks for the help.

**Comment by trangcongloc** [comment]
You can get the name of the thread using [`fetchThreadInfo`](https://fbchat.readthedocs.io/en/stable/api.html#fbchat.Client.fetchThreadInfo)
```py
thr = client.fetchThreadInfo(thread_id)[thread_id]
name = thr.name # get the name of the thread where the message come from
if name == "product" # check if the thread name match the product name
    self.send(Message(text="answer"), thread_id=thread_id, thread_type=thread_type) # Reply when thread name match the product
```
